### ISSUE TITLE
Add Display for the Elements that Violate the Motor Accessibility Guidelines

### ISSUE THREAD

**Issue by hemkan** [hemkan/MotorEase#1]
### Description
User Story: As a user, I should be able to see/know the feature or tool that violates the established guidelines, policies, or standards.

Currently, the accessibility report only show the number of elements that violates the motor accessibility guidelines. This information may not be the most informative for users who wish to improve on their design so this issue focuses on adding information regarding what element of a screen is not meeting the mobile accessibility guidelines.

### Ranking and Reasoning:
This is ranked 1st because we found that this feature would be the most beneficial. This is because it would provide engineers more information on how they could improve and where to start. Also, the ranking of our issues focus on errors > usability/functionality > UI. In this case, my group and I found this issue to be the most beneficial.

### Dependencies
There are no dependencies for this task

### Ideas for Fix
Go through the .xml file for a given screen and find elements/nodes that have clickable set to false. Check and see if those elements relate to the motor accessibility guideline. Compile a list of those elements and cross-check with the AccessiblityReport.txt to see if it's the correct number of elements that violate. From there, display/print a line that writes the names of the elements that are considered violated elements.

### Timeline
Engineering Points/Effort: 2 points (2 full day's worth of work). This bug fix should be completed by Sprint 1.

### Acceptance Criteria.
- Violation Elements has a print/display of those violated in the AccessiblityReport.txt.
- The shown violated elements are accurate upon cross-checking from the .xml file (most likely asking the professor to cross-check to ensure we have the correct understanding).
- Testing will be done with the existing images containing known interactive elements to verify that they are being identified accurately as well new images to ensure it works with all screens.

**Comment by anguyen644** [comment]
After discussing with my teammates, we have decided to transfer this issue over to @GustavoNazarioPerez because I will be working on a another issue #9

**Comment by GustavoNazarioPerez** [comment]
We are transferring to @hemkan because the logic is similar to issue #6 which is assigned to her.
